include/highfive/bits/H5Utils.hpp

```diff
@@ -37,10 +37,15 @@ template <std::size_t N>
 class FixedLenStringArray;
 
 
+template <typename T>
+using unqualified_t = typename std::remove_const<typename std::remove_reference<T>::type
+        >::type;
+
 namespace details {
 template <typename T>
 struct manipulator {
     using type = T;
+    using hdf5_type = unqualified_t<T>;
```

Comment:
to me this is a `base_type` from a composite type. Is there any connection to hdf5?

Reply:
hdf5_type is the type than HDF5 want to see internally.
the type behind the `void*` in `write(void* value)`

Reply:
I don't remember right now when, but sometimes it differs from the `h5_type` (futur) that is the type to store the array.

Reply:
but base_type can be a good name.